packages/accessibility/src/AccessibilityManager.ts

```diff
@@ -222,7 +223,8 @@ export class AccessibilityManager
         self.document.removeEventListener('mousemove', this._onMouseMove, true);
         self.addEventListener('keydown', this._onKeyDown, false);
 
-        this.renderer.off('postrender', this.update);
+        // explicit cast to fix typescript 3.9.5 error - "this expression is not callable"
+        (this.renderer as AbstractRenderer).off('postrender', this.update);
```

Comment:
Why is this not callable without an explicit cast?

Reply:
no idea, in typescript 4.something there is no such error

Reply:
This is no longer an error after merging the latest dev branch.